owners.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Would love if we could use the OWNER file format described in https://chromium.googlesource.com/chromium/tools/depot_tools/+/master/owners.py instead of introducing yet another format.